### PR TITLE
Added note for Address cmdlet availability

### PR DESCRIPTION
**Pull request by adrianwells**
Added note for Exchange Online Address cmdlet availability to help clarify how to view existing Exchange Online address objects via PowerShell

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/address-books/address-book-policies/create-an-address-book-policy.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ In Exchange Online, you can only create ABPs in Exchange Online PowerShell.
 
 An ABP requires one global address list (GAL), one offline address book (OAB), one room list, and one or more address lists. To view the available objects, use the **Get-GlobalAddressList**, **Get-OfflineAddressBook**, and **Get-AddressList** cmdlets.
 
+  **Note**: In Exchange Online, these cmdlets are available only in the Address Lists role, and by default, the role isn't assigned to any role groups. To use this cmdlet, add the Address Lists role to a role group (for example, to the Organization Management role group). For more information, see [Add a role to a role group](https://docs.microsoft.com/Exchange/permissions/role-groups#add-a-role-to-a-role-group)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/address-books/address-book-policies/create-an-address-book-policy.md:29]:
```suggestion
 > [!NOTE]
 > In Exchange Online, these cmdlets are available only in the Address Lists role, and by default, the role isn't assigned to any role groups. To use this cmdlet, add the Address Lists role to a role group (for example, to the Organization Management role group). For more information, see [Add a role to a role group](https://docs.microsoft.com/Exchange/permissions/role-groups#add-a-role-to-a-role-group).
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/address-books/address-book-policies/create-an-address-book-policy.md:29]: Please note the period (.) after the link.

File: Exchange/ExchangeOnline/address-books/address-book-policies/create-an-address-book-policy.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ In Exchange Online, you can only create ABPs in Exchange Online PowerShell.
...
+
   **Note**: The room list that's required for an ABP is an address list that specifies rooms (contains the filter `RecipientDisplayType -eq 'ConferenceRoomMailbox'`). It's not a room finder distribution group that you create with the _RoomList_ switch on the **New-DistributionGroup** or **Set-DistributionGroup** cmdlets.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/address-books/address-book-policies/create-an-address-book-policy.md:31]:
```suggestion
 > [!NOTE]
 > The room list that's required for an ABP is an address list that specifies rooms (contains the filter `RecipientDisplayType -eq 'ConferenceRoomMailbox'`). It's not a room finder distribution group that you create with the _RoomList_ switch on the **New-DistributionGroup** or **Set-DistributionGroup** cmdlets.
```

**Pull request merged**